### DISCUSSION TITLE
Issues with recording a RTSP stream

### DISCUSSION THREAD

**Discussion by jkimsis** [bluenviron/mediamtx#3130] [Questions]
### Question

Hello,

I'm using mediaMTX to stream a RTSP stream, and I also want to record it. The problem I'm running into is that the recording doesn't seem to be happening.

I've set record to yes, set the recording path to be /home/user/recordings/%path/%Y-%m-%d_%H-%M-%S-%f and haven't changed any other options. When I start broadcasting from the camera, the syslog shows a line like this:
INF [path live/a] [record] recording 0 tracks ()

Has anyone encountered this?

**Comment by compilebunny** [comment]
Ditto here. Using an old, low-quality ONVIF IP camera.

[Bug report](https://github.com/bluenviron/mediamtx/issues/3162)

[a comment by an automated account is not shown]
